android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -284,45 +363,95 @@ public void onError(String info) {
+        this.context = new WeakReference<>(context);
+        this.remoteDisplayClass = remoteDisplayClass;
+        this.isEncrypted = encrypted;
+        this.majorProtocolVersion = internalInterface.getProtocolVersion().getMajor();
+        if (listOfStreamingRange != null) {
+            this.listOfStreamingRanges = listOfStreamingRange;
+        }
+    }
+//
+//    // regardless of VideoStreamingParameters are specified or not, we should refer to VideoStreamingCapability.
+//        if (majorProtocolVersion >= 5) {
+//        if (internalInterface.getSystemCapabilityManager() != null) {
+//            final VideoStreamingParameters params = ( parameters == null) ? new VideoStreamingParameters() : new VideoStreamingParameters(parameters);
+//            internalInterface.getSystemCapabilityManager().getCapability(SystemCapabilityType.VIDEO_STREAMING, new OnSystemCapabilityListener() {
+//                @Override
+//                public void onCapabilityRetrieved(Object capability) {
+//                    params.update((VideoStreamingCapability) capability, vehicleMake);    //Streaming parameters are ready time to stream
+//                    startStreaming(params, encrypted);
+//                }
+
+    private void processCapabilitiesWithPendingStart(boolean encrypted, VideoStreamingParameters parameters){
+        if (parameters == null) {
+            if (majorProtocolVersion >= 5) {
+                if (internalInterface.getSystemCapabilityManager() != null) {
+                    internalInterface.getSystemCapabilityManager().getCapability(SystemCapabilityType.VIDEO_STREAMING, new OnSystemCapabilityListener() {
+                        @Override
+                        public void onCapabilityRetrieved(Object capability) {
+                            VideoStreamingParameters params = new VideoStreamingParameters();
+                            VideoStreamingCapability castedCapability = ((VideoStreamingCapability)capability);
+                            VideoStreamManager.this.originalCapability = castedCapability;
+
+                            params.update(castedCapability, vehicleMake);    //Streaming parameters are ready time to stream
+                            VideoStreamManager.this.parameters = params;
+
+                            if (listOfStreamingRanges != null) {
+                                // filtering
+                                castedCapability.setAdditionalVideoStreamingCapabilities(
+                                        getSupportedCapabilities(
+                                                listOfStreamingRanges,
+                                                castedCapability.getAdditionalVideoStreamingCapabilities()
+                                        )
+                                );
+                            } else { }
```

Comment:
empty else statement should be removed
```suggestion
                            }
```